src/Elm/Parser/Declarations.elm

```diff
@@ -21,18 +23,38 @@ declaration : Parser State (Node Declaration)
 declaration =
     oneOf
         [ infixDeclaration
-        , function
-        , typeDefinition
         , portDeclaration
+        , maybeDocumentation
```

Comment:
Do you know if we have tests that parse an port declaration with and without a module documentation?

I don't think `infix` declarations can have module documentation (it's done through the function they alias, weirdly)

Reply:
There are some tests for undocumented  port declarations but we don't have any test for a port with documentation.
It'll make more sense to add one when we support it.

Reply:
Right. What concerns me a bit now is that this becomes a parsing error, so I'd like to make sure that isn't the case.